README.md

```diff
@@ -55,6 +55,24 @@ and its upstream project, [Open Data Hub](https://opendatahub.io/).
    * The wrapper script has several arguments and you can find details in the dedicated document file. See [run_args.md](ods_ci/docs/RUN_ARGUMENTS.md)
    * As alternative, you can run any of the test cases by creating the python virual environment, install the packages in [poetry.lock](poetry.lock) and running the `robot` command directly
 
+# Test tiers
+  - Smoke
+    - Tests executed as part of productization pipeline to verify basic RHOAI functionality
+    - Tests should cover only critical functionality, verifying that build is usable and can be accepted for testing
+    - Tests should be ideally 100% stable and reliable
+    - Test execution time should be less than 1 minute
+  - Sanity
+    - Base test coverage making sure that core functionality works as expected
+    - Tests should be ideally 100% stable and reliable
+    - Test execution time should be less than 5 minutes
+  - Tier1
+    - Test coverage making sure that overall functionality works as expected
+    - Test execution time should be less than 10 minutes
```

Comment:
Can we (later) also document a decision what to do if the time budget for tier1 is exhausted but there are still tier1 tests to be written?

Reply:
I understand it that each test has to be less than 10 minutes but the execution time of the whole set can be longer. Because otherwise we don't match at all currently, right? :upside_down_face:

Reply:
We certainly don't, that's why I was worried these targets might be a bit too ambitious.

Anyways, we should then have timebounds for each test and for suite as a whole; both would provide useful guidance.

Reply:
Yes, the time limit should be related to single test, will adjust the text.

Reply:
Yeah, I agree.

Also, these facts are reason I don't want to approve this myself, since I don't have the historic knowledge and what we should write in here. I agree that being able to follow what is written here would be ideal. But waiting for guys with more history on this project to express themselves :slightly_smiling_face:

Reply:
Adjusted test time description, I hope it is more understandable :)

Reply:
The 2 mandatory approval votes here are ofc. insufficient, this is something that needs to be "socialized to the whole team", to use this phrase I recently saw for the first time. I meant my approval vote more like an approval of the effort which I think is helpful guidance, not as a "merge it" stamp.

Reply:
This PR has been updated just to have a link to the style guide doc. We'll discuss more about the max time per test in the Automatin Meeting